azure-proto-core-test/ResourceGroupOperationsTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+
+namespace azure_proto_core_test
+{
+    public class ResourceGroupOperationsTests
+    {
+        [TestCase]
+        public void CreateResourceFromModel()
+        {
+            //TODO: 4622 needs complete with a Mocked example to fill in this test
+            //public ArmResponse<TOperations> CreateResource<TContainer, TOperations, TResource>(string name, TResource model, azure_proto_core.Location location = default)
```

Comment:
Why not call CreateResource, CreateResourceFromModel? I had to read the code a bit to figure out that this is the use case for these methods.

Reply:
Lets take this in a separate issue since this PR didn't introduce the naming of those libraries.  There will be an in depth naming review during arch board.